OpenRA.Game/Settings.cs

```diff
@@ -20,26 +20,66 @@ namespace OpenRA
 {
 	public enum MouseScrollType { Disabled, Standard, Inverted }
 
+	public class LaunchArgs
```

Comment:
Yeah well, I don't care as long as I don't have to move, redo and rebase stuff again. It's been over 2 months again. I agree that manually setting LaunchSettings to `null` wasn't elegant. Should be FIXED now.